client/state/action-types.js

```diff
@@ -216,8 +216,8 @@ export const EDITOR_SAVE = 'EDITOR_SAVE';
 export const EDITOR_EDIT_RAW_CONTENT = 'EDITOR_EDIT_RAW_CONTENT';
 export const EDITOR_INIT_RAW_CONTENT = 'EDITOR_INIT_RAW_CONTENT';
 export const EDITOR_RESET_RAW_CONTENT = 'EDITOR_RESET_RAW_CONTENT';
-export const EDITOR_TYPE_FETCH = 'EDITOR_TYPE_FETCH';
-export const EDITOR_TYPE_RECEIVE = 'EDITOR_TYPE_RECEIVE';
```

Comment:
Re: naming-wise receive is very ambiguous since it typically means dump the API request in and 🤷‍♀ 

It doesn't always work, but if you can make an optimistic client action whose name makes sense, it's the better naming pattern. Which matches in this case for `EDITOR_TYPE_SET`. I can set the editor type in the local dev console, but it doesn't really make sense to "receive" anything.